### PR TITLE
update CentOS 7 related description

### PR DESCRIPTION
**Pull request by Oreoxmt**
### What is changed, added, or deleted? (Required)

update CentOS 7 related description

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [ ] v1.6 (TiDB Operator 1.6 versions)
- [ ] v1.5 (TiDB Operator 1.5 versions)
- [ ] v1.4 (TiDB Operator 1.4 versions)
- [ ] v1.3 (TiDB Operator 1.3 versions)
- [ ] v1.2 (TiDB Operator 1.2 versions)

### What is the related PR or file link(s)?

- This PR is translated from: 
- Other reference link(s):

### CODE REVIEW

**Review by csuzhangxc — APPROVED** [review]

File: en/deploy-on-gcp-gke.md [hunk 1]
```diff
@@ -235,8 +235,8 @@ The LoadBalancer created for your TiDB cluster is an intranet LoadBalancer. You
 ```shell
 gcloud compute instances create bastion \
     --machine-type=n1-standard-4 \
-    --image-project=centos-cloud \
-    --image-family=centos-7 \
+    --image-project=TODO \
+    --image-family=TODO \
```
Comment (Oreoxmt) [inline en/deploy-on-gcp-gke.md:239]: @csuzhangxc PTAL

Comment (Oreoxmt) [inline en/deploy-on-gcp-gke.md:239]:
```suggestion
```

File: zh/deploy-on-gcp-gke.md [hunk 2]
```diff
@@ -226,8 +226,8 @@ tidb-tikv-2                       1/1     Running   0          47h
 ```shell
 gcloud compute instances create bastion \
     --machine-type=n1-standard-4 \
-    --image-project=centos-cloud \
-    --image-family=centos-7 \
+    --image-project=TODO \
+    --image-family=TODO \
```
Comment (Oreoxmt) [inline zh/deploy-on-gcp-gke.md:230]:
```suggestion
```

**Review by csuzhangxc — APPROVED** [review]

**Pull request merged**